tests/x11/network/NM_wpa2_enterprise.pm

```diff
@@ -36,6 +38,7 @@ sub run {
         assert_and_click 'network_manager-wrong_card_selected';
         assert_screen 'network_manager-network_connected';
     }
+    assert_and_click 'network_manager-popup-cancel';
```

Comment:
Hello @zhaocongjia 
One minor comment here, can you please add some description here as well?
then everyone can know why we don't need to click connect button any more.

LGTM

Reply:
I saw there's no popup window for gnome43 but only a needle with tag "network_manager-popup-cancel" there which seems not real. Can we handle this more elegantly?
https://openqa.opensuse.org/tests/2861036#step/NM_wpa2_enterprise/34

Reply:
I think current way handles both GNOME43 and older GNOME versions.  Otherwise there will be `check_screen`.

Reply:
Yeah, I know it works for all GNOME versions. But it looks weird to assert_and_click a "fake" needle.

Reply:
Hi, I have update code logic to remove the need of `network_manager-popup-cancel` -- now we simply `wait_still_screen` to wait for NetworkManager to auto reconnect.

https://openqa.opensuse.org/tests/2865773
https://openqa.opensuse.org/tests/2865779

Reply:
Great idea. LGTM.